src/utils/api/index.ts

```diff
@@ -1,2 +1,2 @@
-export * from './hooks'
-export * from './metacom'
+export * from '@/hooks/hooks'
```

Comment:
You can remove hooks export from that file, it is not needed

Reply:
Ok, it's better choice

Reply:
No way, `src/app/page.tsx(5,10): error TS2305: Module '"@/utils"' has no exported member 'useApi'.`

Reply:
Wait, I'd rewrite some imports

Reply:
Try to rewrite, it is possible to get rid of that unnecessary export and import useApi hook directly from `@/hooks`